src/events/crawler/scrapers/USA/HI/index.js

```diff
@@ -0,0 +1,75 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as geography from '../../../lib/geography.js';
+import * as parse from '../../../lib/parse.js';
+import * as transform from '../../../lib/transform.js';
+
+const scraper = {
+  country: 'USA',
+  state: 'HI',
+  priority: 1,
+  url: 'https://health.hawaii.gov/docd/advisories/novel-coronavirus-2019/',
+  source: {
+    name: 'Hawaii Department of Health - Disease Outbreak Control Division',
+    url: 'https://health.hawaii.gov/docd/advisories/novel-coronavirus-2019/'
+  },
+  maintainers: [
+    {
+      name: 'Jordan Holt',
+      email: 'jordholt@gmail.com',
+      github: 'Jord-Holt',
+      country: 'USA',
+      flag: '🇺🇸'
+    }
+  ],
+  aggregate: 'county',
+  async scraper() {
+    const counties = [];
+    const $ = await fetch.page(this.url);
+    const $table = $('*:contains("Novel Coronavirus in Hawaii")').closest('table');
+    const $trs = $table.find('tr');
+
+    let stateDeaths = 0;
+    let rowCellValueList;
+
+    $trs.each((index, tr) => {
+      const $tr = $(tr);
+
+      /* 
+          TODO: This range will need to be expanded out when/if a case is found in Kalawao.
+          At time of writing, this county has no cases and is not shown in the data table.
+          Very likely to change. Additionally, index for state deaths will also need to be modified
+          when/if final county is added to table (deaths are only listed for the state and not per county row).
+        */
+      if (index > 2 && index < 7) {
+        // This is to exclude extraneous empty rows in table html.
+
+        rowCellValueList = $tr.text().split('\n');
+
+        const county = rowCellValueList[1].trim();
```

Comment:
Would it be possible to check if the county name is one of Hawaii's counties rather than relying on the index value? That way, the scraper will be more resilient to the addition of new islands.

Reply:
Great idea. I was trying to find a way to avoid using the index and I think this will do nicely. Ill get this changed shortly!